internal/daemon/plugin/vendorplugin.go

```diff
@@ -134,7 +145,6 @@ func WithPathManager(pathManager utils.PathManager) func(*GrpcPlugin) {
 func WithVsp(template_vars VspTemplateVars) func(*GrpcPlugin) {
 	return func(d *GrpcPlugin) {
 		d.vsp = template_vars
-		d.log.Info("Setting VSP", "vsp", d.vsp.VendorSpecificPluginImage)
```

Comment:
Are some of the logs too verbose? Maybe consider having them as V(2) logs that are debug by default but the log level can trigger them to appear. I see another deletion of the log function in line 157 as well.